lib/util/nist_types.ts

```diff
@@ -0,0 +1,35 @@
+export interface CastVoteRecordReport {
+  CVR: CVR
+}
+
+export interface CVR {
+  CurrentSnapshotId: string
```

Comment:
Careful, this, and the `ElectionId` seems to return objects like `{ text: 'cvr-001' }` & `{ text: '1' }` respectively, instead of strings.

Reply:
Fixed in #102